### PR TITLE
Fix 3.12.z release gh action

### PR DESCRIPTION
**Pull request by leszko**
Currently, the release GH Action for 3.12.z does not work. From what I see we need to copy `.github/workflows` into `3.12.z` branch to make it work.

### CODE REVIEW

**Review by hasancelik — CHANGES_REQUESTED** [review]
Good catch 👍 I have some minor comments:
- [test folder at master brach](https://github.com/hazelcast/charts/tree/master/test) should be copied into 3.12.z branch as well to run PR-builder successfully:
```
11:31:10 /tmp/jenkins7425242341460065857.sh: line 4: /home/jenkins/jenkins_slave/workspace/helm-charts-pr-builder/test/run-test: No such file or directory
```
- Also we can remove below files from `3.12.z` branch to prevent possible confusions:
https://github.com/hazelcast/charts/blob/3.12.z/run-test
https://github.com/hazelcast/charts/blob/3.12.z/create-pr-at-official-helm-repo
https://github.com/hazelcast/charts/blob/3.12.z/Jenkinsfile

**Review by leszko — COMMENTED** [review]
@hasancelik Thanks for the comments. I addressed them all. PTAL.

File: test/ct.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+remote: origin
+target-branch: master
```
Comment (hasancelik) [inline test/ct.yaml:2]: @leszko Thanks for the changes 👍 one minor change request before merge the PR, `target-branch` used to identify changed charts and check chart versions. So could you use `3.12.z` branch here?

File: test/run-test [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (hasancelik) [inline test/run-test:1]: PR builder failed due to `.. workspace/helm-charts-pr-builder/test/run-test: Permission denied` error, probably you need to give execute permission to the file.

File: test/ct.yaml (hunk 1 above)
Comment (leszko) [reply test/ct.yaml]: ok, updated.

File: test/run-test (hunk 2 above)
Comment (leszko) [reply test/run-test:1]: It should be fine.

**Review by hasancelik — APPROVED** [review]

**Pull request merged**